posts/2022-10-25-22.0.0.11.adoc

```diff
@@ -0,0 +1,440 @@
+---
+layout: post
+title: "Java SE 19 and distributed security cache support in Open Liberty 22.0.0.11"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: Java SE 19 and distributed security cache support in Open Liberty 22.0.0.11 - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.11 provides support for Java SE 19 and distributed security caches which can improve performance and failure recovery.  Also included are improvements to the BELL feature along with CVE and bug fixes.
+blog_description: Open Liberty 22.0.0.11 provides support for Java SE 19 and distributed security caches which can improve performance and failure recovery.  Also included are improvements to the BELL feature along with CVE and bug fixes.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Java SE 19 and distributed security cache support in Open Liberty 22.0.0.11
+Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+Open Liberty 22.0.0.11 provides support for Java SE 19 and distributed security cache.  Java SE 19 includes many new exciting features and changes such as record patterns and virtual threads.  Distributed security cache support allows multiple Liberty servers to share caches via a JCache provider.  
+
+As part of 22.0.0.11, you can now also expose SPI interfaces as BELL services and inject properties into BELL services, which is simpler to use than writing a user feature. This release also includes an important security vulnerability (CVE) fix, as well as many notable bug fixes.
+
+
+In link:{url-about}[Open Liberty] 22.0.0.11:
+
+* <<java19, Java SE 19 Support>>
+* <<security_caches, Distributed security caches>>
+* <<bell, Expose SPI interfaces as BELL services and inject properties into BELL services>>
```

Comment:
Suggestion:
Expose SPI interfaces as BELL services and inject properties into BELL services
-->
Implement SPI interfaces with BELL services and enable BELL services to receive configuration properties